app/controllers/users_controller.rb

```diff
@@ -0,0 +1,6 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.find(params(:id))
```

Comment:
oh shucks. Yes, you're right. That's when I was trying to redirect to users/id after sign in. Either way It should be a bracket enclosing the ```[:id]```